apps/pie-docs/src/assets/styles/components/_indexPage.scss

```diff
@@ -0,0 +1,50 @@
+@use '../fozzie-config' as f;
+@use '../variables' as v;
+@use '@justeattakeaway/pie-css/scss' as p;
+
+.c-indexPage {
```

Comment:
Just noticed at 320px width (minimum we support) it feels a little squashed where there is no inline whitespace

<img width="527" alt="Screenshot 2024-08-30 at 15 51 15" src="https://github.com/user-attachments/assets/74d1f577-b480-4df8-98b2-5375e11f16af">